gallery/src/components/NxSmallTag/NxSmallTagExample.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxSmallTag } from '@sonatype/react-shared-components';
+
+function NxSmallTagExample() {
+  return (
+    <>
+      <NxSmallTag>Default</NxSmallTag>
+      <NxSmallTag color="blue">Blue</NxSmallTag>
+      <NxSmallTag color="indigo">Indigo</NxSmallTag>
+      <NxSmallTag>Wwwwwwwwwwwwwwww</NxSmallTag>
```

Comment:
This happens to produce a tag that is right at its max size with no apparent overflow.  We want to visually test what happens when there _is_ overflow, so please make it a bit longer.  You might want to use text that makes it clear that it's broken on purpose, such as "This tag content is too long and should overflow".

Reply:
Updated here 95a0d8db60872e6bf8ee325930863d30bba6fbe3, be0238cbe083e05e4bafdfc87982cb303fc97a3d.